src/test/java/at/ac/tuwien/kr/alpha/solver/AbstractSolverTests.java

```diff
@@ -100,6 +106,9 @@ void ignoreNonDefaultDomainIndependentHeuristics() {
 		String[] grounders = getProperty("grounders", "naive");
 		String[] stores = getProperty("stores", ci ? "alpharoaming,naive" : "alpharoaming");
 		String[] heuristics = getProperty("heuristics", ci ? "ALL" : "NAIVE,VSIDS");
+		String[] gtcValues = getProperty("grounderToleranceConstraints", "strict,permissive");
+		String[] gtrValues = getProperty("grounderToleranceRules", "strict");
```

Comment:
Would it make sense to run in CI with both, `strict` and `permissive`, setting here?

Reply:
In principle yes, but at the cost of a 100% increase in the number of test cases, and time needed to conduct these tests. I already had `permissive` in here at one point, but chose to remove it again to keep CI time at a reasonable level.